lib/quby/questionnaires/entities/fields.rb

```diff
@@ -21,30 +21,22 @@ class Fields
         # targeted by :depends_on relations.
         attr_reader :input_keys
 
-        def initialize
+        def initialize(questionnaire)
```

Comment:
Dat gegeven bestaat nog niet bij het initialiseren van de questionnaire, aangezien de dsl incrementeel geevalueerd wordt en de fields al in de initialiser van questionnaire geinitialised wordt.